trailblazer/store/api.py

```diff
@@ -200,32 +156,15 @@ def update_run_status(self, analysis_id: int, ssh: bool = False) -> None:
         if analysis.workflow_manager == WorkflowManager.TOWER.value:
             self.update_tower_run_status(analysis_id=analysis_id)
         elif analysis.workflow_manager == WorkflowManager.SLURM.value:
-            self.update_analysis_from_slurm_run_status(analysis_id=analysis_id, ssh=ssh)
+            self.update_analysis_from_slurm_run_status(analysis_id=analysis_id)
 
-    def update_analysis_from_slurm_run_status(self, analysis_id: int, ssh: bool = False) -> None:
-        """Query slurm for entries related to given analysis, and update the analysis in the database."""
-        analysis: Analysis = self.get_analysis_with_id(analysis_id=analysis_id)
+    def update_analysis_from_slurm_run_status(self, analysis_id: int) -> None:
+        """Query slurm for entries related to given analysis, and update the analysis in the database.
+        Raises: Exception if unable to parse SLURM jobs
```

Comment:
We don't raise an error in here, we only seem to catch, log and then set the analysis to Error

Reply:
Removed